config/secret.txt

```diff
@@ -27,8 +27,8 @@ $include private_default.txt
 # IPINTEL_EMAIL ch@nge.me
 
 ##Used to send round data to a webhook. Can now use HTTPS thanks to rust_g
-#WEBHOOK_ADDRESS https://example.com
-#WEBHOOK_KEY webkey
+WEBHOOK_ADDRESS http://127.0.0.1:8443/byond
```

Comment:
```suggestion
WEBHOOK_ADDRESS http://127.0.0.1:8443/byond
```

🤔

Reply:
Shouldnt matter as our secret config is not on there, but yes I agree, unneeded change